### ISSUE TITLE
logged in twice on same stage

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#193]
i am currently on the hentest stage as a logged in player in both firefox and chrome. in the past it has always been that you can't be logged in to the same stage twice - you can enter as audience with another browser window, but not as logged in player.
if it doesn't cause any conflicts then perhaps it's not a big problem, but it would be good to think through the implications of having the same player twice on a stage.

**Comment by vickis** [comment]
yes that could be a complication (or an advantage! for those who like the avatar version of sleight of hand ;) it does seem more likely to be the former to me

**Comment by helenvarley** [comment]
yeah i think it could be confusing. if you were trying to operate 2 avatars, it would be easier just to switch from one to the other in the same browser. you can't operate in both browsers at the same time. the only advantage might be if you needed to share your log-in, e.g. let's say there is a bug that has corrupted someone's account & also made it impossible for an admin to create a new log-in or edit an existing one, & you needed to share a log-in for a performance. but this has never happened before, & would hopefully not happen at the moment of a performance.
i can't think of another situation where it would be useful to be able to share a log in, or be logged in to the same stage at the same time from different browsers (apart from testing, but even then it doesn't save any time really).

**Comment by rohrlach** [comment]
As it stands, each browser will act independently. 

Once I'm done building state into the performance engine, the browsers will mirror each other (Holding an avatar in one, means holding an avatar in the other)

One player, one audience behavior won't change

**Comment by helenvarley** [comment]
ok; i guess we need to monitor this down the track & see whether it's a problem or not.

**Comment by vickis** [comment]
i don’t think it would be useful to be able to share logins
this is bug :)

a big fat lovely weta!
